src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -361,6 +362,9 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
      if raw=true
     :raises: :class:`CloudError<msrestazure.azure_exceptions.CloudError>`
     """
+    if ssh_key_value is not None and not _is_valid_ssh_rsa_public_key(ssh_key_value):
+        raise CLIError('Provided ssh key ({}) is invalid or non-existent'.format(ssh_key_value))
```

Comment:
If it was invalid, then yes.  Seems correct, it's printing the thing that is invalid (and public key's not secret)

If you are worried about line length/wrapping, I can truncate if you prefer.